src/BlockchainTestsFiller/ValidBlocks/bcEIP1559/lowDemandFiller.yml

```diff
@@ -0,0 +1,781 @@
+      extraData: '0x42'
+
+
+  # Block 50
+  - transactions:
+    - data: ''
+      accessList: []
+      gasLimit: 50000000
+      tip: 0x00
+      feeCap: 1000
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xcccccccccccccccccccccccccccccccccccccccc
+      value: 0x0
+
+  # Block 51
+  - blockHeader:
+      extraData: '0x42'
+
+
+  # Block 52
+  - transactions:
+    - data: ''
+      accessList: []
+      gasLimit: 50000000
+      tip: 0x00
+      feeCap: 1000
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xcccccccccccccccccccccccccccccccccccccccc
+      value: 0x0
+
+  
+
+
+#    blockHeader:
+#      gasTarget: 0xffc0000
+
+
+
+
+
+  expect:
+  - network:
+    - ">=London"
+    result:
+      0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC:
+        storage:
+          # These blocks all have the same total gas, so the ratio between
+          # the old gas price and the new one is roughly constant.
+          #
+          0x01: 0x036b # 875 = 1000*.875
+          0x02: 0x02fe # 766 ~= 875*.875
+          0x03: 0x029f # 671 ~= 766*.875
+          0x04: 0x024c # 588 ~= 671*.875 
+          0x05: 0x0203 # 515 ~= 588*.875
+          0x06: 0x01c3 # 451 ~= 515*.875
+          0x07: 0x018b # 395 ~= 451*.875
+          0x08: 0x015a # 346
+          0x09: 0x012f # 303
+          0x0a: 0x010a # 266
+          0x0b: 0x00e9 # 233
+          0x0c: 0x00cc # 204
+          0x0d: 0x00b3 # 179
+          0x0e: 0x009d # 157
+          0x0f: 0x008a # 138
+          0x10: 0x0079 # 121
+          0x11: 0x006a # 106
+          0x12: 0x005d #  93
+          0x13: 0x0052 #  82
+          0x14: 0x0048 #  72
+          0x15: 0x0040 #  64
+          0x16: 0x0039 #  57
+          0x17: 0x0032 #  50
+          0x18: 0x002c #  44
+          0x19: 0x0027 #  39
+          0x1a: 0x0023 #  35
+          0x1b: 0x001f #  31
+          0x1c: 0x001c #  28
+          0x1d: 0x0019 #  25
+          0x1e: 0x0016 #  22
+          0x1f: 0x0014 #  20
+          0x20: 0x0012 #  18
+          0x21: 0x0010 #  16
+          0x22: 0x000f #  15
+          0x23: 0x000e #  14
+          0x24: 0x000d #  13
+          0x25: 0x000c #  12
+          0x26: 0x000b #  11
+          0x27: 0x000a #  10
+          0x28: 0x0009 #   9
+
+          # Once baseFee gets to 8 it stays there, because
+          # the fraction is slightly more than 7/8, since we
+          # use some gas
+          0x29: 0x0008 #   8
+          0x2a: 0x0008 #   8 
+          0x2b: 0x0008 #   8
+          0x2c: 0x0008 #
+          0x2d: 0x0008 #
+          0x2e: 0x0008 #
+          0x2f: 0x0008 #
+          0x30: 0x0008 #
+
+          # If we put a block that doesn't use any gas,
+          # the ratio is exactly 7/8's, so the following block
+          # has a baseFee of 7.
+
+          0x32: 0x0007
+
+          # baseFee doesn't go below 7. 7*7/8 is 6.125, but then
+          # it is rounded up
```

Comment:
this is a very cool test. here you can see the capacity of blockchain format!

Reply:
Yes.